src/irc/core/modes.c

```diff
@@ -743,6 +743,7 @@ static char *get_nicks(IRC_SERVER_REC *server, WI_ITEM_REC *item,
 			    g_hash_table_lookup(optlist, "yes") == NULL) {
                                 /* too many matches */
 				g_string_free(str, TRUE);
+                                g_strfreev(matches);
```

Comment:
While the indentation is a mess (yep, there's another wrong line in the context part of this diff), we try to avoid making the problem worse. Tabs instead of spaces please.